### ISSUE TITLE
Changing the name of a note removes the file

### ISSUE THREAD

**Issue by dktcoding** [Offerel/roundcube_primitivenotes#20]
Hi again!

When writing #19 I found that when the title of a note is modified, the file is lost and converted to the note type.

For instance, if I upload a PDF:

![Screen Shot 2019-04-15 at 17 01 40](https://user-images.githubusercontent.com/709639/56161649-6e36c500-5fa0-11e9-9a32-38daadf91129.png)

And click on "edit", after changing the title, the PDF is gone, and all I have left is an empty HTML with the right title:

![Screen Shot 2019-04-15 at 17 02 08](https://user-images.githubusercontent.com/709639/56161685-8b6b9380-5fa0-11e9-8dbe-14e33f6dba7f.png)

The error log states:

```
[15-Apr-2019 17:01:56 America/Argentina/Buenos_Aires] PHP Warning:  strpos(): Offset not contained in string in /home/*******/public_html/plugins/primitivenotes/notes.php on line 411
```

Regards

**Comment by Offerel** [comment]
I will have a look ath that

**Comment by Offerel** [comment]
Fixed with 4bdacab873093ed01b935829a487de228e81546e
